src/machine/machine_rp2040_usb.go

```diff
@@ -151,7 +151,10 @@ func initEndpoint(ep, config uint32) {
 		usbDPSRAM.EPxBufferControl[ep].Out.SetBits(usbBuf0CtrlAvail)
 
 	case usb.ENDPOINT_TYPE_INTERRUPT | usb.EndpointOut:
-		// TODO: not really anything, seems like...
+		val |= usbEpControlEndpointTypeInterrupt
+		usbDPSRAM.EPxControl[ep].Out.Set(val)
+		usbDPSRAM.EPxBufferControl[ep].Out.Set(USBBufferLen & usbBuf0CtrlLenMask)
+		usbDPSRAM.EPxBufferControl[ep].Out.SetBits(usbBuf0CtrlAvail)
```

Comment:
The following also need to be implemented
However, it would be better to separate them into separate PRs.
I will try to work on atsamd51 .

* samd21
* samd51
* nrf52840